src/plugins/intel_npu/src/plugin/npuw/partitioning/online/snapshot.cpp

```diff
@@ -44,12 +44,44 @@ bool isOp(const std::shared_ptr<ov::Node>& node) {
     }
     return true;
 }
+
+std::vector<std::string> getWeightsPrecision(const std::shared_ptr<ov::Node>& node) {
```

Comment:
The order here might be incorrect. With the particular model we get a split of 4 new repeated blocks due to u4/u8 precision. My guess was that it'll be split into 2 new blocks.